service/pokemon.go

```diff
@@ -22,6 +26,115 @@ func (sp PokemonService) GetPokemonByID(pokemons []model.Pokemon, ID int) (model
 
 }
 
+// GetPokemonWorker: Get Pokemons based on ID (odd or even) using Workers pools. 
+func (sp PokemonService)GetPokemonWorker(pokemons []model.Pokemon, t string, items int, itemsPerWorker int) ([]model.Pokemon){
+	var wg sync.WaitGroup 
+	retrievePokemons:=  make([]model.Pokemon,0)
```

Comment:
I want to initialize an empty slice. So I initialize it with 0 "cap".  I know this may be a little confused, so I will updated to only bee an empty slice(model.Pokemon[])